zio-sql/jvm/src/main/scala/Sql.scala

```diff
@@ -461,6 +461,15 @@ trait Sql {
 
   type :||:[A, B] = Features.Union[A, B]
 
+  def exprName[F, A, B](expr: Expr[F, A, B]): Option[String] =
+    expr match {
+      case Expr.Source(_, c) => Some(c.name)
+      case _ => None
+    }
+
+  implicit def expToSelection[F, A, B](expr: Expr[F, A, B]): Selection[F, A, SelectionSet.Cons[A, B, SelectionSet.Empty]] =
```

Comment:
Can you place this into the companion object of `Selection`? It should be found there anyway, so you shouldn't need to import it, but it will keep it tidier than placing at the top level.

Reply:
Ok, no problem, but then it will be necessary to import Selection._ (see TestBasicSelect)
```scala
val userSql = new Sql { self =>
    import self.ColumnSet._
    import self.Selection._
...
```